megamek/src/megamek/server/GameManager.java

```diff
@@ -33555,6 +33564,10 @@ public void addReport(Report report) {
         vPhaseReport.addElement(report);
     }
 
+    public void addAllReport(Vector<Report> reports) {
```

Comment:
I'm all for extracting methods but this doesn't seem to add anything over just writing the addAll in place of the method call, or does it?

Reply:
not really was just making it similar to and not accessing vPhaseReport directly

    public void addReport(Report report) {
        vPhaseReport.addElement(report);
    }